<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerCart</class>
 <widget class="QWidget" name="CustomerCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomerCart</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Item&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Customer Cart&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CustomerCartSubmit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CustomerCartItem">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Quantity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CustomerCartAdd">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CustomerCartCategory">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>121</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="CustomerCartQuantity">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowItems">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Items</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Package Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Package Quantity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="customerPackageQuantity">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>CustomerCartSubmit</sender>
   <signal>clicked()</signal>
   <receiver>CustomerCart</receiver>
   <slot>BillingPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomerCartAdd</sender>
   <signal>clicked()</signal>
   <receiver>CustomerCart</receiver>
   <slot>addCartSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ShowItems</sender>
   <signal>clicked()</signal>
   <receiver>CustomerCart</receiver>
   <slot>showItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomerCartItem</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>CustomerCart</receiver>
   <slot>showPackageQuantity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BillingPage()</slot>
  <slot>addCartSubmit()</slot>
  <slot>showItems()</slot>
  <slot>showPackageQuantity()</slot>
 </slots>
</ui>
